FEATURES.md

```diff
@@ -56,3 +56,8 @@ for a detailed explanation.
 
   - `interaction.<event-name>` for events handled by a component.
   - `interaction.ember-action` for closure actions.
+
+* `ember-runtime-enumerable-contains`
+
+Deprecates `Enumerable.contains` and `Array.contains` in favor of `Enumerable.includes` and `Array.includes` 
```

Comment:
Again here I lean toward `Enumerable#contains`, per https://github.com/emberjs/website/pull/2600#discussion_r64891064. It is a Rubyish approach, curious if there are other thoughts.